fedimg/utils.py

```diff
@@ -128,17 +132,27 @@ def unxz_source_file(file_path):
     ])
 
     if retcode != 0:
-        return ''
+        _log.error("Unable to unxz the image: (output) %s,"
+                   "(error) %s, (return_code) %s" % (
+                       output, error, retcode))
```

Comment:
The above can be simplified if we just print the error output above in the external_run_command function